examples/offline_inference/data_parallel.py

```diff
@@ -64,6 +64,18 @@ def parse_args():
     parser.add_argument(
         "--trust-remote-code", action="store_true", help="Trust remote code."
     )
+    parser.add_argument(
+        "--max-num-seqs",
+        type=int,
+        default=64,
+        help=("Maximum number of sequences to be processed in a single iteration."),
+    )
```

Comment:
Are both of these args required to avoid the OOM? 64 is quite small for batch mode, would be good if we could fix just with the `gpu_memory_utilization` reduction...

Reply:
yeah, also seems that we need much more memory during initialization than before. I was about to investigate more into this, but didn't get time to do so. Wondering if @yewentao256 could dig further into this?